### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by sven1103**
1.9.0 (2021-07-19)
------------------

**Added**

* Introduce ``life.qbic.utils.MaxQuantParser`` to validate the filestructure resulting from a maxQuant run  #60 

**Fixed**

* Ensures, that the BioinformaticAnalysisParser throws only exceptions as stated in the DataSetParser interface #62)

**Dependencies**

**Deprecated**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
     <dependency>
       <artifactId>data-model-lib</artifactId>
       <groupId>life.qbic</groupId>
-      <version>2.7.0</version>
+      <version>2.10.0-SNAPSHOT</version>
```
Comment (Steffengreiner) [inline pom.xml:90]: This should be changed to 
```suggestion
      <version>2.10.0</version>
```
as soon as the data-model-lib 2.10.0 is released. 
I'd argue that this blocks this release until DML is released

File: pom.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <description>Collection of non-Vaadin, non-Liferay utilities.</description>
   <modelVersion>4.0.0</modelVersion>
   <name>Core Utilities Library</name>
-  <version>1.8.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
+  <version>1.9.0-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
```
Comment (Steffengreiner) [inline pom.xml:11]: The SNAPSHOT Tag will be removed automatically right?

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
LGTM! However, I'm unsure on how to proceed concerning the data-model-lib dependency.

File: pom.xml (hunk 2 above)
Comment (sven1103) [reply pom.xml:11]: yes exactly

File: pom.xml (hunk 1 above)
Comment (sven1103) [reply pom.xml:90]: ups, you are right

**Review by Steffengreiner — APPROVED** [review]
Nice! 🚀

**Pull request merged**